README.md

```diff
@@ -30,7 +30,7 @@ the port is merged upstream.
   ```python3
   import io
   import threading
-  import yaml
+  import yaml_ft
```

Comment:
I think it should be `import yaml_ft as yaml` in the README too. Maybe even with a try/except to fall back to importing `yaml`?

Also IMO the try/except deserves some discussion in the readme.

Reply:
Agreed.